compiler/rustc_borrowck/src/diagnostics/conflict_errors.rs

```diff
@@ -262,8 +263,8 @@ impl<'tcx> MirBorrowckCtxt<'_, '_, '_, 'tcx> {
                         format!(
                             "consider creating a fresh reborrow of {} here",
                             self.describe_place(moved_place)
-                                .map(|n| format!("`{n}`"))
-                                .unwrap_or_else(|| "the mutable reference".to_string()),
+                                .map(|n| Cow::Owned(format!("`{n}`")))
+                                .unwrap_or_else(|| "the mutable reference".into()),
```

Comment:
as I understand it using `format_args!` would avoid allocation even in the Some case:
```suggestion
                                .map(|n| format_args!("`{n}`"))
                                .unwrap_or_else(|| format_args!("the mutable reference")),
```